docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Troubleshooting
+description: >-
+  Learn how to troubleshoot common configuration issues or work around any outstanding bugs.
+sidebarDepth: 0
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium troubleshooting faq frequently asked questions
+---
+
+# Troubleshooting
+
+This article provides troubleshooting information for various tools and features in Pomerium.
+
+[[toc]]
+
+## Pomerium Core
+
+### HSTS
+
+By default, Pomerium sends the [`Strict-Transport-Security`](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) response header to the browser, which pins the certificate to our browser for one year. This is common best practice to help prevent man-in-the-middle attacks, but can create issues while a new Pomerium configuration is in development.
+
+When you visit an endpoint while Pomerium is using the self-signed bootstrap certificate or a Let's Encrypt staging certificate (before switching to a production certificate), the untrusted certificate may be pinned in your browser. It then must be reset once Pomerium is switched to production certificates.
+
+While developing your Pomerium environment, consider adjusting the [`SET_RESPONSE_HEADERS`](/reference/readme.md#set-response-headers) key to remove `Strict-Transport-Security` or reduce the `max-age` value until your production certificates are in place.
+
+See [this article](https://www.ssl2buy.com/wiki/how-to-clear-hsts-settings-on-chrome-firefox-and-ie-browsers) for more information on clearing HSTS for specific endpoints across common browsers.
+
+### JWT Authentication
+
+When securing the Pomerium Authenticate service with a certificate signed by Let's Encrypt, your upstream applications may reject the certificate when attempting to access the JWT signing key. Here's an example log line from [Grafana](/guides/grafana.md):
+
+```log
+logger=context error=Get "https://authenticate.localhost.pomerium.io/.well-known/pomerium/jwks.json": x509: certificate signed by unknown authority
+```
+
+This is often due to the recent expiration of the [DST Root CA X3](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/) certificate. Many default keystores used by docker images and less-frequently updated distributions still carry this expired certificate. Even though LE certs are cross-signed with the [ISRG Root X1](https://letsencrypt.org/certificates/) CA certificate, some applications will still reject them.
```

Comment:
```suggestion
This is often due to the recent expiration of the [DST Root CA X3](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/) certificate. Many default keystores used by docker images and less-frequently updated distributions still carry this expired certificate. Even though Let's Encrypt (LE) certs are cross-signed with the [ISRG Root X1](https://letsencrypt.org/certificates/) CA certificate, some applications will still reject them.
```

If we don't re-insert Let's Encrypt in the above, let's define it before using the LE acronym.